modules/task/migrate.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package task
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"runtime"
+	"strings"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/structs"
+	"code.gitea.io/gitea/modules/util"
+)
+
+func handleCreateError(owner *models.User, err error, name string) error {
```

Comment:
I agree. However, errors are special in that they should not be taken lightly: the format string needs to have the correct placeholders ('%d', '%s', etc.). If not for security reasons, at least to prevent a bigger failure; an i18n solution for this should validate the user format string and use a hard-coded default if the check fails.
Anyway, that's for another PR.